Core/Parser/SchemaParser.cs

```diff
@@ -340,14 +341,17 @@ public async Task<BebopSchema> Parse()
                 _errors.Add(e);
             }
 
-            var isReadOnly = Eat(TokenKind.ReadOnly);
+            var modifiers = RecordModifiers.None;
+            if (Eat(TokenKind.ReadOnly)) modifiers |= RecordModifiers.ReadOnly;
+            if (Eat(TokenKind.Mut)) modifiers |= RecordModifiers.Mut;
```

Comment:
It maybe doesn't super matter right now, but if we ever get more record modifiers that are realistically stackable, we could do something like
```suggestion
            var modifiers = RecordModifiers.None;
            while (true)
            {
                if (Eat(TokenKind.ReadOnly)) modifiers |= RecordModifiers.ReadOnly;
                else if (Eat(TokenKind.Mut)) modifiers |= RecordModifiers.Mut;
                else break;
            }
```
to allow specifying them in any order.